README.md

```diff
@@ -4,3 +4,5 @@ Chrysalis bundle for Keyboard.io
 A heavy work in progress. Don't expect anything fancy. Or friendly. Yet.
 
 To launch the development environment, simply type `yarn && yarn start`.
+
+Please make sure you've got the latest firmware from https://github.com/keyboardio/Model01-Firmware (at least v1.22 is required for communication with your keyboard)
```

Comment:
```suggestion
Please make sure you have a firmware with [bi-directional communication][plugin:FocusSerial] and [EEPROMKeymap][plugin:eepromkeymap] enabled. The latest beta (v1.93-beta) of the [factory firmware][factory-firmware], or its latest master branch might be a good starting point.

 [plugin:FocusSerial]: https://github.com/keyboardio/Kaleidoscope/blob/master/doc/plugin/FocusSerial.md
 [plugin:eepromkeymap]: https://github.com/keyboardio/Kaleidoscope/blob/master/doc/plugin/EEPROM-Keymap.md
 [factory-firmware]: https://github.com/keyboardio/Model01-Firmware
```

Reply:
v1.22 is not enough, nor is v1.23. The required functionality is only in the latest beta (v1.93). I think it also makes sense to highlight that one does not need to use the factory firmware, as long as the appropriate plugins are enabled. But the factory firmware is a good starting point.

That's the reasoning behind the suggested changes :)